index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mfadhili Gitau</title>
+    <link rel="stylesheet" hrefv="./css/main.css">
```

Comment:
You seem not to have linked your CSS properly by making a mistake with the name for the href attribute. Kindly correct this from `hrefv` and not `href`